src/client/components/general/Searchbar/searchbar.scss

```diff
@@ -0,0 +1,27 @@
+@use '/src/client/assets/styles/styles' as s;
+@import '../../../assets/styles/variables';
+
+.search {
+    position: relative;
+    
+    .search-icon {
+        margin-right: 15px;
+    }
+
+    input {
+        height: 40px;
+        font-size: 1em;
+        line-height: 45px;
+        padding-left: 10px;
+        border: 2px solid $thirdColor;
+        width: 300px;
+        outline: none;
+        background: #fff; 
+        color: #a2a5a7;
+        margin-right: 12px;
+        position: absolute;
+        z-index: 15;
+        top: -15px;
```

Comment:
Maybe we can take a look next week on how to improve this position absolute we have here and play around with visibility: 'hidden' instead

Reply:
Sure! :)
Also for productDisplay or is it ok to be absolute?

Reply:
For ProductDisplay is okay to be absolute, because you have conditional rendering to render that component. You're not using something like `top: -25px ` so the component is 'hidden' outside the viewport. 
You can use `position: absolute`, this is very used, to be honest. But there are some cases where this is not the best solution, specially if you have to hide the component outside the viewport, that's the only thing